setup.py

```diff
@@ -29,8 +29,8 @@
 ]
 
 setup(
-    name='django-haystack',
-    version='2.5.1-rgg-1',
+    name='django-haystack-rgg-fork',
```

Comment:
renaming the package so we can use a semantic version without colliding with the official package name